admin/package-files/flocker-control-agent.firewalld.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I would be more inclined to split things up by tool (so `firewalld/flocker-control-agent.xml`), if we were to split things up.